mongoimport/csv.go

```diff
@@ -153,12 +153,14 @@ func (c CSVConverter) Convert() (b bson.D, err error) {
 		c.useArrayIndexFields,
 	)
 	if _, ok := err.(coercionError); ok {
-		c.Print()
+		if err = c.Print(); err != nil {
+			return
```

Comment:
I don't think so. This function is using named return values. I'm not a fan of this, but this diff is already huge, so I wanted to avoid optional changes.